templates/workspace_services/guacamole/user_resources/guacamole-azure-export-reviewvm/windowsvm.diff

```diff
@@ -1,6 +1,6 @@
 diff -Naur ../guacamole-azure-windowsvm/terraform/data.tf terraform/data.tf
```

Comment:
This is a bit hard to read in the diff as-is, I suggest go to 3 dot menu -> View file